ubiconfig/utils/config_schema.json

```diff
@@ -22,35 +22,38 @@
             "required": ["name", "stream"]
             }
     },
-    "content_sets": {
-        "debuginfo": {"$ref": "#/definitions/content_sets_base"},
-        "rpm": {"$ref": "#/definitions/content_sets_base"},
-        "srpm": {"$ref": "#/definitions/content_sets_base"}
-    },
-    "arches": {
-        "type": "array",
-        "items": {"type": "string"}
-    },
-    "modules": {
-        "type": "object",
-        "properties": {
-            "include": {
-                "type": "array",
-                "items":{"$ref": "#/definitions/module"}
-                }
-            }
+    "properties":{
+        "content_sets": {
+            "debuginfo": {"$ref": "#/definitions/content_sets_base"},
+            "rpm": {"$ref": "#/definitions/content_sets_base"},
+            "srpm": {"$ref": "#/definitions/content_sets_base"}
         },
-    "packages": {
-        "type": "object",
-        "properties": {
-            "include": {
-                "type": "array",
-                "items": {"type":"string"}
-                },
-            "exclude": {
-                "type": "array",
-                "items": {"type": "string"}
+        "arches": {
+            "type": "array",
+            "items": {"type": "string"}
+        },
+        "modules": {
+            "type": "object",
+            "properties": {
+                "include": {
+                    "type": "array",
+                    "items":{"$ref": "#/definitions/module"}
+                    }
+                }
+            },
+        "packages": {
+            "type": "object",
+            "properties": {
+                "include": {
+                    "type": "array",
+                    "items": {"type":"string"}
+                    },
+                "exclude": {
+                    "type": "array",
+                    "items": {"type": "string"}
+                    }
                 }
             }
-        }
+        },
+    "additionalProperties": false
```

Comment:
This is a bit inconsistent that additionalProperties: false is only set here on the top-level object.  If the intent is to be strict and only allow correct property names, shouldn't all the sub-objects like modules, packages etc also have that?

Reply:
Makes sense. I'll do it